### PR TITLE
add schema for yield aggregator

### PR DESCRIPTION
**Pull request by 0xroll**
https://github.com/delta-hq/schemas/issues/33

changes : 
adding schema for yield agg and ALM
moving the rest to `Others`

### CODE REVIEW

File: schemas/automated-liquidity-management/schema.json [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+          "description": "The symbol of the receipt token.",
+          "type": "string"
+        }
+      }
+    },
+    {
+      "label": "Position Snapshot",
+      "tableName": "automated_liquidity_depositors",
+      "aggregation": "daily",
+      "uniqueKey": ["user_address", "pool_address", "block_date"],
```
Comment (melotik) [inline schemas/automated-liquidity-management/schema.json]:
```suggestion
      "uniqueKey": ["user_address", "underlying_liquidity_pool_address", "block_date", "token_index"],
```

File: schemas/automated-liquidity-management/schema.json [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+        },
+        "strategy_vault_contract_address": {
+          "description": "The address of the strategy vault this user has a position in.",
+          "type": "string"
+        },
+        "user_address": {
+          "description": "The address of the user who has a position in the strategy vault.",
+          "type": "string"
+        },
+        "underlying_liquidity_pool_address": {
```
Comment (melotik) [inline schemas/automated-liquidity-management/schema.json]: Can we call this `liquidity_pool_address` or `pool_address` same with the pools table? Underlying seems redundant to say, isn't there only one liquidity pool address?

File: schemas/automated-liquidity-management/schema.json [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+          "description": "The address of the underlying liquidity pool where liquidity are deposited into",
+          "type": "string"
+        },
+        "underlying_token_address": {
```
Comment (melotik) [inline schemas/automated-liquidity-management/schema.json:79]: Again, not sure if we need to say underlying in all of this? Can probably drop `underlying_` right?

File: schemas/automated-liquidity-management/schema.json [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+          },
+          "event_type": {
+              "description": "The type of event, corresponds to the action taken by the user (ie, deposit, withdrawal).",
+              "type": "string"
+          }
+      }
+  },
+  {
+    "label": "Incentive Claim Data",
+    "tableName": "misc_incentive_claim",
```
Comment (melotik) [inline schemas/automated-liquidity-management/schema.json]: do we need to name this?
```suggestion
    "tableName": "automated_liquidity_incentive_claim",
```

File: schemas/automated-liquidity-management/schema.json [hunk 5]
```diff
@@ -0,0 +1,253 @@
...
+        },
+        "total_fees_usd": {
+          "description": "The total amount of revenue and fees paid in this pool in the given snapshot, in USD.",
+          "type": "number"
+        }
+      }
+    },
+    {
+      "label": "Events",
+      "tableName": "liquidity_yield_events",
```
Comment (melotik) [inline schemas/automated-liquidity-management/schema.json]:
```suggestion
      "tableName": "automated_liquidity_events",
```

??

File: schemas/automated-liquidity-management/schema.json (hunk 3 above)
Comment (0xroll) [reply schemas/automated-liquidity-management/schema.json:79]: there is a receipt token given to users that deposit in ALMs . so maybe lets keep this

**Comment by 0xroll** [comment]
`automated_liquidity_events` will need some changes, but i think we can merge and show to the teams first

**Review by melotik — APPROVED** [review]

**Comment by melotik** [comment]
> `automated_liquidity_events` will need some changes, but i think we can merge and show to the teams first

@0xroll if you know the changes, I would try to add them in now. But not a blocker imo, lgtm

**Pull request merged**